lib/vmdb/settings/hash_differ.rb

```diff
@@ -19,6 +19,8 @@ def self.diff(h1, h2)
           child =
             if v1.kind_of?(Hash) && v2.kind_of?(Hash)
               diff(v1, v2)
+            elsif v1.kind_of?(Array) && v2.kind_of?(Array)
+              v2 + (v1 - v2).map { |x| "#{KNOCKOUT_STRING}#{x}" }
```

Comment:
I don't think this should live in the HashDiffer as that class should not be aware of the config gem (it's generic diffing code)...let's discuss today if you have time.